From aab68dcdf5193802394d7264680453426ecbf9df Mon Sep 17 00:00:00 2001
From: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
Date: Tue, 26 Sep 2023 16:56:33 +0530
Subject: [PATCH] New backport upgrade to 6.5-rc3 from 6.1.24 version

Added code changes for successful backport build without compilation
issue. Disabling the qcom QMI helper from new backport version.

Signed-off-by: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
---
 Makefile.kernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.kernel b/Makefile.kernel
index 7071d14..4386f47 100644
--- a/Makefile.kernel
+++ b/Makefile.kernel
@@ -39,7 +39,7 @@ obj-y += compat/
 
 obj-$(CPTCFG_CFG80211) += net/wireless/
 obj-$(CPTCFG_MAC80211) += net/mac80211/
-obj-$(CPTCFG_QCOM_QMI_HELPERS) += drivers/soc/qcom/
+#obj-$(CPTCFG_QCOM_QMI_HELPERS) += drivers/soc/qcom/
 obj-$(CPTCFG_WLAN) += drivers/net/wireless/
 obj-$(CPTCFG_USB_NET_RNDIS_WLAN) += drivers/net/usb/
 
-- 
2.17.1

